datageneration/tests/test_retrieve_combination.py

```diff
@@ -10,7 +10,7 @@
 
 class TestCombinationRetriever(unittest.TestCase):
     def setUp(self):
-        self.retriever = CombinationRetriever(source='datageneration/tests/data/Primary_Keys_test.xlsx',
+        self.retriever = CombinationRetriever(source='datageneration/tests/data/Primary_Keys_filtered10.xlsx',
```

Comment:
maybe strict with test in the name

Reply:
I can rename it, both the file and the in code. Please add the test file to the git commit next time, then I can simply use whatever is there! :)